.github/workflows/ci.yml

```
@@ -174,6 +174,10 @@ jobs:
         run: pnpm run test:spellcheck-coverage
         background: true
 
+      - name: Run ensure-engine-built tests
+        run: pnpm run test:ensure-engine-built
+        background: true
```

actionlint (1.7.12)

[error] unexpected key "background" for step to run shell command. expected one of "continue-on-error", "env", "id", "if", "name", "run", "shell", "timeout-minutes", "working-directory"

(syntax-check)